### PR TITLE
Remove stanford renderer completely!

### PR DESCRIPTION
**Pull request by caesar2164**
This PR needs to wait until AFTER Carnegie courses sunset (early November 2016)

### CODE REVIEW

**Comment by stvstnfrd** [comment]
We get a big coverage bump here, as we weren't testing this one..

File: gen_cert.py [hunk 1]
```diff
@@ -465,230 +464,6 @@ def _use_unicode_font(self, string):
...
-
-        #  Student name
-        # default is to use the DejaVu font for the name, will fall back
-        # to Arial if there are unusual characters
-        style = styleOpenSansLight
-        style.fontSize = 34
-        width = stringWidth(student_name.decode('utf-8'), 'OpenSans-Bold', style.fontSize) / mm
-        paragraph_string = "<b>{0}</b>".format(student_name)
-
-        if self._use_unicode_font(student_name):
```
Comment (stvstnfrd) [inline gen_cert.py:601]: You should double-check, but I think this is the final remaining reference to this method.
We should be able to remove it at this point.

Comment (stvstnfrd) [reply gen_cert.py:601]: And once that goes, so can `_contains_characters_above`.

**Comment by caesar2164** [comment]
sad, I only get 66% now because I remove v3...

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

**Comment by stvstnfrd** [comment]
> sad, I only get 66% now because I remove v3...

Well, that PR (v3) dropped as low as 60%, so all things relative :)

**Review by stvstnfrd — APPROVED** [review]

**Comment by stvstnfrd** [comment]
@caesar2164 Does this need rebased now?

**Pull request merged**